cryptol-remote-api/python/cryptol/__init__.py

```diff
@@ -212,7 +212,7 @@ def process_result(self, res : Any) -> Any:
                     for m in res['models']
                     for arg in m]
         else:
-            raise ValueError("Unknown sat result " + str(res))
+            raise ValueError("Unknown prove or sat result " + str(res))
```

Comment:
Should this mention "`safe`" as well here?

Reply:
Perhaps it should just say "Unknown SMT result"? I.e., something more general that describes the underlying dependency/failure instead of naming all of our API functions that utilize it...?

Reply:
Sounds good to me. I especially like the idea of having fewer places to remember to keep in sync with the rest of the implementation.